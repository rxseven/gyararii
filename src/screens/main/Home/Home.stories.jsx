import React from 'react';
import { storiesOf } from '@storybook/react';

import Home from './index';

const title = 'Screens/Main/Home';
const component = <Home />;

storiesOf(title, module).add('default', () => component);
