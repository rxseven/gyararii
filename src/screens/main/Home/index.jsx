import React from 'react';

import { Body, Document, Head } from 'components/common/base/Page';
import Layout from 'components/common/composite/Layout';
import Gallery from 'components/common/modules/Gallery';

function Home() {
  return (
    <Document>
      <Head>
        <title>ギャラリー</title>
        <meta
          content="A simple React app for collecting photos"
          name="description"
        />
      </Head>
      <Body>
        <Layout>
          <Gallery />
        </Layout>
      </Body>
    </Document>
  );
}

export default Home;
