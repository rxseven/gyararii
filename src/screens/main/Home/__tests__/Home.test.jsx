import { shallow } from 'enzyme';
import React from 'react';

import Home from '../index';

describe('<Home />', () => {
  // Arrange
  const component = <Home />;

  describe('Unit tests', () => {
    it('should render without crashing', () => {
      // Act
      const wrapper = shallow(component);

      // Assert
      expect(wrapper).toBeDefined();
    });
  });
});
