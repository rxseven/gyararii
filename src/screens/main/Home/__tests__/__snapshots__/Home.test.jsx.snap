// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> Snapshot tests should render correctly 1`] = `
<div
  className="row justify-content-sm-center"
>
  <div
    className="col"
  >
    <div
      className="sc-hMqMXs cjGUwN"
    >
      <div
        className="sc-jKJlTe ifsCBM"
      >
        <div
          className="container"
        >
          <div
            className="row"
          >
            <div
              className="col"
            >
              <div
                className="sc-ckVGcZ cuDuMa"
              >
                <label
                  className="sc-gqjmRU egfDhJ"
                  htmlFor="gallery"
                >
                  <span
                    className="btn btn-primary btn-sm disabled"
                    data-place="right"
                    data-tip="Please DO NOT upload inappropriate images."
                    disabled={true}
                    role="button"
                  >
                    <span
                      className="sc-htpNat iYntvg"
                    >
                      <svg
                        aria-hidden="true"
                        className="svg-inline--fa fa-cloud-upload-alt fa-w-20 "
                        data-icon="cloud-upload-alt"
                        data-prefix="fas"
                        focusable="false"
                        role="img"
                        style={Object {}}
                        viewBox="0 0 640 512"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <path
                          d="M537.6 226.6c4.1-10.7 6.4-22.4 6.4-34.6 0-53-43-96-96-96-19.7 0-38.1 6-53.3 16.2C367 64.2 315.3 32 256 32c-88.4 0-160 71.6-160 160 0 2.7.1 5.4.2 8.1C40.2 219.8 0 273.2 0 336c0 79.5 64.5 144 144 144h368c70.7 0 128-57.3 128-128 0-61.9-44-113.6-102.4-125.4zM393.4 288H328v112c0 8.8-7.2 16-16 16h-48c-8.8 0-16-7.2-16-16V288h-65.4c-14.3 0-21.4-17.2-11.3-27.3l105.4-105.4c6.2-6.2 16.4-6.2 22.6 0l105.4 105.4c10.1 10.1 2.9 27.3-11.3 27.3z"
                          fill="currentColor"
                          style={Object {}}
                        />
                      </svg>
                    </span>
                    <span
                      className="sc-bxivhb iGoLHq"
                    >
                      Upload
                    </span>
                    <span
                      className="sc-bwzfXH iDiJqN"
                    >
                      <span>
                        <style>
                           @-webkit-keyframes brsk-three-bounce { 0%, 80%, 100% { -webkit-transform: scale(0); } 40% { -webkit-transform: scale(1); } } @keyframes brsk-three-bounce { 0%, 80%, 100% { transform: scale(0); -webkit-transform: scale(0); } 40% { transform: scale(1); -webkit-transform: scale(1); } } 
                        </style>
                        <div
                          style={
                            Object {
                              "WebkitAnimationDelay": "-0.32s",
                              "WebkitAnimationDuration": "1.25s",
                              "WebkitAnimationIterationCount": "infinite",
                              "WebkitAnimationName": "brsk-three-bounce",
                              "WebkitAnimationTimingFunction": "ease-in-out",
                              "animationDelay": "-0.32s",
                              "animationDuration": "1.25s",
                              "animationIterationCount": "infinite",
                              "animationName": "brsk-three-bounce",
                              "animationTimingFunction": "ease-in-out",
                              "backgroundColor": "#fff",
                              "borderRadius": "100%",
                              "display": "inline-block",
                              "height": 7,
                              "marginRight": 5,
                              "width": 7,
                            }
                          }
                        />
                        <div
                          style={
                            Object {
                              "WebkitAnimationDelay": "-0.16s",
                              "WebkitAnimationDuration": "1.25s",
                              "WebkitAnimationIterationCount": "infinite",
                              "WebkitAnimationName": "brsk-three-bounce",
                              "WebkitAnimationTimingFunction": "ease-in-out",
                              "animationDelay": "-0.16s",
                              "animationDuration": "1.25s",
                              "animationIterationCount": "infinite",
                              "animationName": "brsk-three-bounce",
                              "animationTimingFunction": "ease-in-out",
                              "backgroundColor": "#fff",
                              "borderRadius": "100%",
                              "display": "inline-block",
                              "height": 7,
                              "marginRight": 5,
                              "width": 7,
                            }
                          }
                        />
                        <div
                          style={
                            Object {
                              "WebkitAnimationDuration": "1.25s",
                              "WebkitAnimationIterationCount": "infinite",
                              "WebkitAnimationName": "brsk-three-bounce",
                              "WebkitAnimationTimingFunction": "ease-in-out",
                              "animationDuration": "1.25s",
                              "animationIterationCount": "infinite",
                              "animationName": "brsk-three-bounce",
                              "animationTimingFunction": "ease-in-out",
                              "backgroundColor": "#fff",
                              "borderRadius": "100%",
                              "display": "inline-block",
                              "height": 7,
                              "width": 7,
                            }
                          }
                        />
                      </span>
                    </span>
                  </span>
                  <input
                    className="sc-gZMcBi jOngK"
                    disabled={true}
                    id="gallery"
                    multiple={true}
                    onChange={[Function]}
                    type="file"
                  />
                </label>
                <div
                  className="sc-gzVnrw liWgEh"
                />
              </div>
            </div>
          </div>
          <div
            className="sc-eNQAEJ bnYtcV"
          />
        </div>
      </div>
      <div
        className="sc-kgoBCf eIUqfm"
      >
        <div
          className="card-columns"
        >
          <div />
        </div>
      </div>
      <div
        className="sc-dxgOiQ fjhBbd"
        hidden={false}
      >
        <span>
          <style>
             @-webkit-keyframes brsk-three-bounce { 0%, 80%, 100% { -webkit-transform: scale(0); } 40% { -webkit-transform: scale(1); } } @keyframes brsk-three-bounce { 0%, 80%, 100% { transform: scale(0); -webkit-transform: scale(0); } 40% { transform: scale(1); -webkit-transform: scale(1); } } 
          </style>
          <div
            style={
              Object {
                "WebkitAnimationDelay": "-0.32s",
                "WebkitAnimationDuration": "1.25s",
                "WebkitAnimationIterationCount": "infinite",
                "WebkitAnimationName": "brsk-three-bounce",
                "WebkitAnimationTimingFunction": "ease-in-out",
                "animationDelay": "-0.32s",
                "animationDuration": "1.25s",
                "animationIterationCount": "infinite",
                "animationName": "brsk-three-bounce",
                "animationTimingFunction": "ease-in-out",
                "backgroundColor": "#999",
                "borderRadius": "100%",
                "display": "inline-block",
                "height": 7,
                "marginRight": 5,
                "width": 7,
              }
            }
          />
          <div
            style={
              Object {
                "WebkitAnimationDelay": "-0.16s",
                "WebkitAnimationDuration": "1.25s",
                "WebkitAnimationIterationCount": "infinite",
                "WebkitAnimationName": "brsk-three-bounce",
                "WebkitAnimationTimingFunction": "ease-in-out",
                "animationDelay": "-0.16s",
                "animationDuration": "1.25s",
                "animationIterationCount": "infinite",
                "animationName": "brsk-three-bounce",
                "animationTimingFunction": "ease-in-out",
                "backgroundColor": "#999",
                "borderRadius": "100%",
                "display": "inline-block",
                "height": 7,
                "marginRight": 5,
                "width": 7,
              }
            }
          />
          <div
            style={
              Object {
                "WebkitAnimationDuration": "1.25s",
                "WebkitAnimationIterationCount": "infinite",
                "WebkitAnimationName": "brsk-three-bounce",
                "WebkitAnimationTimingFunction": "ease-in-out",
                "animationDuration": "1.25s",
                "animationIterationCount": "infinite",
                "animationName": "brsk-three-bounce",
                "animationTimingFunction": "ease-in-out",
                "backgroundColor": "#999",
                "borderRadius": "100%",
                "display": "inline-block",
                "height": 7,
                "width": 7,
              }
            }
          />
        </span>
      </div>
      <div
        onClick={[Function]}
        style={
          Object {
            "bottom": 0,
            "cursor": "pointer",
            "left": 0,
            "opacity": 0,
            "position": "absolute",
            "right": 0,
            "transitionDelay": "0s",
            "transitionDuration": "0.2s",
            "transitionProperty": "opacity, visibility",
            "transitionTimingFunction": "linear",
            "visibility": "hidden",
          }
        }
      >
        <svg
          aria-hidden="true"
          className="svg-inline--fa fa-arrow-alt-circle-up fa-w-16 sc-VigVT hvKqCR"
          data-icon="arrow-alt-circle-up"
          data-prefix="fas"
          focusable="false"
          role="img"
          style={Object {}}
          viewBox="0 0 512 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M8 256C8 119 119 8 256 8s248 111 248 248-111 248-248 248S8 393 8 256zm292 116V256h70.9c10.7 0 16.1-13 8.5-20.5L264.5 121.2c-4.7-4.7-12.2-4.7-16.9 0l-115 114.3c-7.6 7.6-2.2 20.5 8.5 20.5H212v116c0 6.6 5.4 12 12 12h64c6.6 0 12-5.4 12-12z"
            fill="currentColor"
            style={Object {}}
          />
        </svg>
      </div>
    </div>
  </div>
</div>
`;
